### PR TITLE
FIX for int arrays in ep.eventtype

### PR DESCRIPTION
**Pull request by jasmainak**
closes https://github.com/mne-tools/mne-python/issues/4564

@agramfort can you live without a test here? :)

### CODE REVIEW

File: mne/io/eeglab/eeglab.py [hunk 1]
```diff
@@ -510,7 +510,8 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
                 if isinstance(ep.eventtype, int):
                     ep.eventtype = str(ep.eventtype)
```
Comment (jona-sassenhagen) [inline mne/io/eeglab/eeglab.py]: Hm ... can't we lose this line now?

Comment (jasmainak) [reply mne/io/eeglab/eeglab.py]: if I understand correctly `eventtype` can be an array or even just a single element. This condition checks for the case of a single element

**Comment by agramfort** [comment]
+1 for MRG when green.

**Comment by jona-sassenhagen** [comment]
The Travis failure is just 1 build timing out, I have restarted it. The others are green.

**Comment by larsoner** [comment]
Thanks @jasmainak

**Comment by jona-sassenhagen** [comment]
Thanks!

**Pull request merged**